plugins/chat/app/serializers/chat/message_serializer.rb

```diff
@@ -57,7 +57,7 @@ def user
     end
 
     def excerpt
-      object.censored_excerpt
+      object.excerpt || object.raw_excerpt
```

Comment:
we don't censor the second one in this case no?

Reply:
I might be confusing myself a bit here, but strangely we already used the cooked value within raw_excerpt (from before this pr), which has already applied censoring of watched words.

Reply:
I will just apply censoring again from the serializer for now anyway

Reply:
Kind of strange, but previously we were double censoring excerpts. Since we have watched-words as part of our features override array which gets passed into PrettyText.cook, it means that our excerpt will already be censored. So we don't actually need to apply it a second time. I've updated this approach to prevent this.

Reply:
Ah yes I came to the same conclusion in another comment, seems right :+1: